airflow/www/app.py

```diff
@@ -81,7 +81,9 @@ def create_app(config=None, testing=False):
     flask_app.secret_key = conf.get("webserver", "SECRET_KEY")
 
     flask_app.config["PERMANENT_SESSION_LIFETIME"] = timedelta(minutes=settings.get_session_lifetime_config())
-    flask_app.config.from_pyfile(settings.WEBSERVER_CONFIG, silent=True)
+    # Enable customizations in webserver_config.py to be applied via Flask.current_app.
```

Comment:
Wondering if we should make this a configurable option.

The interested folks can enable it and the rest may choose not to

Reply:
Given it would already be a choice to import `flask.current_app` or not in *webserver_config.py*, I am not sure what value an extra configurable option step would add?

Reply:
The import is basically free and does not change anything, but setting up the app context is not. That might sound backwards but is how Flask works.

In this instance though, the heavy lifting has already been done before this line, and I can’t think of things that you can do only when the context is _not_ enabled, so a config doesn’t sound that useful to me.